rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/docker/DockerImageNameFactory.groovy

```diff
@@ -13,8 +13,12 @@ class DockerImageNameFactory extends ImageNameFactory {
 
   @Override
   def buildImageName(BakeRequest bakeRequest, List<PackageNameConverter.OsPackageName> osPackageNames) {
+    String imageName = bakeRequest.extended_attributes?.get('docker_target_image_name')
```

Comment:
I know the name isn't perfect, but doesn't specifying `ami_name` accomplish the same thing as you're doing with `docker_target_image_name`?